compns_stochy.F90

```diff
@@ -294,7 +294,7 @@ subroutine compns_stochy (me,sz_nml,input_nml_file,fn_nml,nlunit,deltim,iret)
             'land perturbations will be applied to selected paramaters, using newer scheme designed for DA ens spread'
                do k =1,n_var_lndp
                    select case (lndp_var_list(k))
-                   case('vgf','smc','stc','alb', 'sal','emi') 
+                   case('vgf','smc','stc','alb', 'sal','emi','zol','zoi','zow') 
```

Comment:
Should this be just `zol` for the time being? I see that the code will stop anyway in `lndp_apply_perts`, so I think either way is fine.

Reply:
Yes, it should. Sorry, forgot to remove them.

Reply:
@DomHeinzeller Yes, it could be a very good change to allocate arrays conditionally.
I'll appreciate if you could do the ufs-weather-model PR. Thank you.

Reply:
Can you please make the change in `compns_stochy.F90` so that it reads and then push it to this PR?
```
case('vgf','smc','stc','alb','sal','emi','zol')
```

Reply:
Thanks!